sirepo/package_data/template/controls/parameters.py.jinja

```diff
@@ -154,26 +156,28 @@ def read_cols_and_save_report(default_file, save_file, cols_file):
         os.rename(default_file, save_file)
 
 def update_and_run_simulation(correctors):
-    is_success = run_simulation(correctors)
-    readings = []
-    columns = madx_parser.parse_tfs_file('twiss.file.tfs')
-    for idx in range(len(columns.name)):
-        keyword = columns.keyword[idx].replace('"', '')
-        if keyword == 'MONITOR':
-            readings += [float(columns.x[idx]), float(columns.y[idx])]
-        elif keyword == 'HMONITOR':
-            readings += [float(columns.x[idx])]
-        elif keyword == 'VMONITOR':
-            readings += [float(columns.y[idx])]
-    res = np.array(readings)
-    if is_success:
+    global finished_sims
```

Comment:
Unnecessary. You only need `global` for variables you are assigning. Modifications to globals don't need `global`.

That being said, why isn't finished_sims just a flag? It seems like the only time it is interesting is in the append(1) case. If it ever hits, then set it to True, and that's success. In this case, `global` is required.

Reply:
good point. Switched it to flag with global